salt/key.py

```diff
@@ -309,12 +295,9 @@ def __init__(self, opts):
         # Remove the key attribute set in KeyCLI.__init__
```

Comment:
It looks class `MultiKeyCLI` is not needed anymore.

Reply:
You're right that it's not being used in Salt code anymore. I wasn't sure if I should remove that here in case other people were using it outside. But it's probably best to not maintain this since it's part of the raet support.